support/mock/mockrt.go

```diff
@@ -648,6 +687,17 @@ func (rt *Runtime) ExpectVerifyPoSt(post abi.WindowPoStVerifyInfo, result error)
 	}
 }
 
+func (rt *Runtime) ExpectVerifyConsensusFault(h1, h2, extra []byte, result *runtime.ConsensusFault, resultErr error) {
+	rt.expectVerifyConsensusFault = &expectVerifyConsensusFault{
+		requireCorrectInput: true,
+		BlockHeader1:        h1,
+		BlockHeader2:        h2,
+		BlockHeaderExtra:    extra,
+		Fault:               result,
+		Err:                 resultErr,
+	}
+}
+
 // Verifies that expected calls were received, and resets all expectations.
 func (rt *Runtime) Verify() {
```

Comment:
Add checks for the new expectations for `DeleteActor` and `VerifyConsensusFault` to ensure the expected calls arrived.